internal/locate/region_request.go

```diff
@@ -1040,6 +1044,14 @@ func (s *RegionRequestSender) SendReqCtx(
 			return resp, nil, err
 		}
 
+		var isLocalTraffic bool
+		if staleReadCollector != nil && s.replicaSelector != nil {
+			if target := s.replicaSelector.targetReplica(); target != nil {
+				isLocalTraffic = target.store.IsLabelsMatch(s.replicaSelector.labels)
```

Comment:
If there's no label set, is the logic working as expected?

Reply:
If no label set, `isLocalTraffic` will always be true. All traffic and qps will be local. I think this is expected.